src/NzbDrone.Core/Tv/RefreshSeriesService.cs

```diff
@@ -146,20 +146,26 @@ private List<Season> UpdateSeasons(Series series, Series seriesInfo)
 
         public void Execute(RefreshSeriesCommand message)
         {
-            _eventAggregator.PublishEvent(new SeriesRefreshStartingEvent(message.Trigger == CommandTrigger.Manual));
+            var manualTrigger = message.Trigger == CommandTrigger.Manual;
+            _eventAggregator.PublishEvent(new SeriesRefreshStartingEvent(manualTrigger));
 
             if (message.SeriesId.HasValue)
             {
                 var series = _seriesService.GetSeries(message.SeriesId.Value);
-                RefreshSeriesInfo(series);
+
+                // Only refresh automatically if the triggering manually or refreshed more than 1 hour ago
+                if (manualTrigger || series.LastInfoSync < DateTime.UtcNow.AddHours(-1))
```

Comment:
Given the earlier comments I don't think th is code is needed. But if you want to keep it, then use _checkIfSeriesShouldBeRefreshed instead.